### PR TITLE
Feature/sergeyro/tooltip on keyboard focus

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/3949004433

### CODE REVIEW

File: src/components/BreadcrumbsBar/BreadcrumbItem/BreadcrumbItem.tsx [hunk 1]
```diff
@@ -8,8 +8,8 @@ import { HideShowEvent } from "../../../constants";
 import { SubIcon, VibeComponentProps } from "../../../types";
 import "./BreadcrumbItem.scss";
 
-const MOUSEENTER = [HideShowEvent.MOUSE_ENTER];
-const MOUSELEAVE = [HideShowEvent.MOUSE_LEAVE];
+const SHOW_TRIGGERS = [HideShowEvent.FOCUS, HideShowEvent.MOUSE_ENTER];
```
Comment (orrgottlieb) [inline src/components/BreadcrumbsBar/BreadcrumbItem/BreadcrumbItem.tsx:11]: this is redundant and not intuitive, should be a default for mouse_enter/mouse_leave, mouse_over

File: src/components/ButtonGroup/ButtonWrapper.tsx [hunk 2]
```diff
@@ -6,6 +6,9 @@ import Tooltip from "../Tooltip/Tooltip";
 import { MoveBy } from "../../types/MoveBy";
 import { DialogPosition } from "../../constants/positions";
 
+const SHOW_TRIGGERS = [HideShowEvent.FOCUS, HideShowEvent.MOUSE_ENTER];
```
Comment (orrgottlieb) [inline src/components/ButtonGroup/ButtonWrapper.tsx:9]: same comment

**Review by orrgottlieb — CHANGES_REQUESTED** [review]
The change should be in the infra of the Dialog, as it should map the relevant event to the corresponding keyboard events.

please reconsider the implementation

**Review by hadasfa — COMMENTED** [review]
let's do here what we discuss - we will talk about it in our meeting

**Pull request closed without merging**